ChaosMod/Components/EffectDispatcher.cpp

```diff
@@ -409,13 +409,13 @@ void EffectDispatcher::UpdateMetaEffects(int deltaTime)
 		std::vector<std::tuple<EffectIdentifier, EffectData *>> availableMetaEffects;
 
 		float totalWeight = 0.f;
-		for (auto &[effectId, effectData] : g_EnabledEffects)
+		for (auto effectData : GetFilteredEnabledEffects())
```

Comment:
```suggestion
		for (auto &effectData : GetFilteredEnabledEffects())
```
So this doesn't potentially bite us in the future should GetFilteredEnabledEffects not return references anymore for whatever reason.